simphony_mayavi/cuds/vtk_lattice.py

```diff
@@ -175,16 +261,18 @@ def from_lattice(cls, lattice):
             y, z, x = numpy.meshgrid(
                 range(size[1]), range(size[2]), range(size[0]))
             indices = izip(x.ravel(), y.ravel(), z.ravel())
-        elif lattice_type == 'Hexagonal':
-            x, y, z = numpy.meshgrid(
-                range(size[0]), range(size[1]), range(size[2]))
+        elif lattice_type in BravaisLattice:
```

Comment:
compared to the `if` part, it is not clear what kind of lattice_type is processed by this block. Does it imply only hexagonal like in the old code?